test/e2e/test_job_submissions.py

```diff
@@ -2015,6 +2016,215 @@ def session_action_has_expected_progress(session_action_id) -> bool:
+                                "hostRequirements": {
+                                    "attributes": [
+                                        {
+                                            "name": "attr.worker.os.family",
+                                            "allOf": [os.environ["OPERATING_SYSTEM"]],
+                                        }
+                                    ]
+                                },
+                                "script": {
+                                    "actions": {"onRun": {"command": "whoami"}},
+                                },
+                            }
+                        ],
+                    }
+                )
+            )
+
+        config = configparser.ConfigParser()
+        set_setting("defaults.farm_id", deadline_resources.farm.id, config)
+        set_setting("defaults.queue_id", queue_to_use.id, config)
+        job_id: Optional[str] = api.create_job_from_job_bundle(
+            job_bundle_path,
+            job_parameters,
+            priority=99,
+            config=config,
+            queue_parameter_definitions=[],
+        )
+
+        assert job_id is not None
+
+        job_details = Job.get_job_details(
+            client=deadline_client,
+            farm=deadline_resources.farm,
+            queue=queue_to_use,
+            job_id=job_id,
+        )
+
+        LOG.info(f"job details: {job_details}")
+        assert job_details.get("task_run_status") == "SUSPENDED"
+        attachments: Optional[dict] = job_details.get("attachments")
+        assert attachments is not None
+
+        manifests: list[dict[str, Any]] = attachments["manifests"]
+
+        assert manifests is not None
+        first_manifest = manifests[0]
+
+        input_manifest_path = first_manifest["inputManifestPath"]
+
+        # Find the input manifest
+        queue_job_attachment_settings: dict[str, Any] = deadline_client.get_queue(
+            farmId=deadline_resources.farm.id,
+            queueId=queue_to_use.id,
+        )["jobAttachmentSettings"]
+
+        job_attachments_bucket_name: str = queue_job_attachment_settings["s3BucketName"]
+        root_prefix: str = queue_job_attachment_settings["rootPrefix"]
+
+        s3_client = boto3.client("s3")
+
+        get_manifest_object_result: dict[str, Any] = s3_client.get_object(
+            Bucket=job_attachments_bucket_name,
+            Key=root_prefix + "/Manifests/" + input_manifest_path,
+        )
+
+        get_object_result_body: dict[str, Any] = json.loads(
+            get_manifest_object_result["Body"].read()
+        )
+
+        # Get the Job Attachment bucket file paths of the input files
+        input_file_paths: list[dict[str, Any]] = get_object_result_body["paths"]
+        first_input_file_hash = input_file_paths[0]["hash"]
+
+        # Delete one of the input files from the Job Attachments bucket after confirming that it exists
+
+        s3_client.get_object(
```

Comment:
[HeadObject](https://docs.aws.amazon.com/AmazonS3/latest/API/API_HeadObject.html) is a lighter API call if we just want to check for existence, but if the role doesn't have permission to call it already, don't worry about it.